packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -6,7 +6,8 @@ import { withTheme } from 'emotion-theming';
 import { setValue, clearValues } from '@appbaseio/reactivecore/lib/actions';
 import { componentTypes, CLEAR_ALL } from '@appbaseio/reactivecore/lib/utils/constants';
 import types from '@appbaseio/reactivecore/lib/utils/types';
-import { getClassName, handleA11yAction } from '@appbaseio/reactivecore/lib/utils/helper';
+import { getClassName, handleA11yAction, isEqual } from '@appbaseio/reactivecore/lib/utils/helper';
+import XDate from 'xdate';
```

Comment:
Not sure if you have compared library. But this one is also a nice one:
https://date-fns.org/

Reply:
Yes, we compare it. The `xdate` library is lighter than `date-fns`.